LunaMary/src/Quiz.java

```diff
@@ -0,0 +1,23 @@
+import java.util.List;
+
+
+class Quiz {
+
+    private List<IQuestion> iQuestions;
+    private TypeQuiz typeQuiz;
+//    private String ageRange;
```

Comment:
ageRange should be an Integer attribute which will save the minimum age to take this survey.